### ISSUE TITLE
Turtlebot4 does not move in Ignition Fortress

### ISSUE THREAD

**Issue by aharshac** [turtlebot/turtlebot4#443]
### Robot Model

Select One

### ROS distro

Select One

### Networking Configuration

Select One

### OS

Select One

### Built from source or installed?

Select One

### Package version

Installed by running `sudo apt install ros-humble-turtlebot4-simulator` which also installed the following dependencies:

Package|Version
-- | --
ros-humble-ackermann-steering-controller | 2.35.0-1jammy.20240611.150636
ros-humble-admittance-controller | 2.35.0-1jammy.20240611.152045
ros-humble-bicycle-steering-controller | 2.35.0-1jammy.20240611.150635
ros-humble-effort-controllers | 2.35.0-1jammy.20240611.150523
ros-humble-force-torque-sensor-broadcaster | 2.35.0-1jammy.20240611.150216
ros-humble-forward-command-controller | 2.35.0-1jammy.20240611.150246
ros-humble-ign-ros2-control | 0.7.9-1jammy.20240702.133731
ros-humble-imu-sensor-broadcaster | 2.35.0-1jammy.20240611.150756
ros-humble-irobot-create-common-bringup | 2.1.0-1jammy.20240611.152747
ros-humble-irobot-create-control | 2.1.0-1jammy.20240611.152548
ros-humble-irobot-create-ignition-bringup | 2.1.0-1jammy.20240702.134602
ros-humble-irobot-create-ignition-toolbox | 2.1.0-1jammy.20240611.143003
ros-humble-irobot-create-nodes | 2.1.0-1jammy.20240611.143008
ros-humble-position-controllers | 2.35.0-1jammy.20240611.150535
ros-humble-range-sensor-broadcaster | 2.35.0-1jammy.20240611.145853
ros-humble-ros2-controllers | 2.35.0-1jammy.20240611.152436
ros-humble-steering-controllers-library | 2.35.0-1jammy.20240611.150247
ros-humble-tricycle-controller | 2.35.0-1jammy.20240611.145926
ros-humble-tricycle-steering-controller | 2.35.0-1jammy.20240611.150639
ros-humble-turtlebot4-ignition-bringup | 1.0.2-1jammy.20240702.134808
ros-humble-turtlebot4-simulator | 1.0.2-1jammy.20240702.135349
ros-humble-velocity-controllers | 2.35.0-1jammy.20240611.150527

Ended up afterwards with:
![Turtlebot4 sim dependencies](https://github.com/user-attachments/assets/dedf0ef2-c96b-4b76-a60f-8272a7afcdc7)

### Type of issue

Select One

### Expected behaviour

The simulator is started with `ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py`.

### Actual behaviour

The robot oscillates in place on the dock for a moment after pressing the start simulation button. 

Undock function does not work:

> [motion_control-35] [INFO] [1721156313.253158752] [motion_control]: Received new undock goal
> [turtlebot4_node-30] [ERROR] [1721156313.253800347] [turtlebot4_node]: undock goal was rejected by server
> [turtlebot4_node-30] [ERROR] [1721156313.253949995] [turtlebot4_node.rclcpp_action]: unknown goal response, ignoring...
> [turtlebot4_node-30] [ERROR] [1721156313.254056785] [turtlebot4_node.rclcpp_action]: unknown goal response, ignoring...

When the back button is pressed on the teleop widget:
> [motion_control-35] [WARN] [1721156116.986160734] [motion_control]: Reached backup limit! Stop Driving robot backward or disable from safety_override parameter!

There's a different error when rviz arg is set true. Maybe I'll open another ticket.

### Error messages

```bash
[motion_control-35] [INFO] [1721157458.998124552] [motion_control]: Enabling REFLEX_BUMP
[motion_control-35] [INFO] [1721157458.998509529] [motion_control]: Enabling REFLEX_CLIFF
[motion_control-35] [INFO] [1721157458.998636966] [motion_control]: Enabling REFLEX_STUCK
[motion_control-35] [INFO] [1721157458.998701344] [motion_control]: Enabling REFLEX_WHEEL_DROP
[wheel_status_publisher-36] [INFO] [1721157458.985583435] [wheel_status_publisher]: Advertised topic: wheel_vels
[wheel_status_publisher-36] [INFO] [1721157458.986167474] [wheel_status_publisher]: Advertised topic: wheel_ticks
[mock_publisher-37] [INFO] [1721157459.012922126] [mock_publisher]: Advertised mocked topic: slip_status
[static_transform_publisher-44] [WARN] [1721157459.073658549] []: Old-style arguments are deprecated; see --help for new-style arguments
[robot_state-38] [INFO] [1721157459.097406718] [robot_state]: Advertised topic: battery_state
[robot_state-38] [INFO] [1721157459.100081280] [robot_state]: Subscription to topic: dock_status
[robot_state-38] [INFO] [1721157459.106364610] [robot_state]: Advertised topic: stop_status
[robot_state-38] [INFO] [1721157459.107760765] [robot_state]: Subscription to topic: odom
[static_transform_publisher-45] [WARN] [1721157459.109421028] []: Old-style arguments are deprecated; see --help for new-style arguments
[static_transform_publisher-44] [INFO] [1721157459.111098200] [rplidar_stf]: Spinning until stopped - publishing transform
[static_transform_publisher-44] translation: ('0.000000', '0.000000', '0.000000')
[static_transform_publisher-44] rotation: ('0.000000', '0.000000', '0.000000', '1.000000')
[static_transform_publisher-44] from 'rplidar_link' to 'turtlebot4/rplidar_link/rplidar'
[kidnap_estimator_publisher-39] [INFO] [1721157459.134587064] [kidnap_estimator_publisher]: Advertised topic: kidnap_status
[kidnap_estimator_publisher-39] [INFO] [1721157459.135397097] [kidnap_estimator_publisher]: Subscription to topic: hazard_detection
[ui_mgr-40] [INFO] [1721157459.142048077] [ui_mgr]: Subscription to topic: cmd_lightring
[ui_mgr-40] [INFO] [1721157459.144334123] [ui_mgr]: Subscription to topic: cmd_audio
[static_transform_publisher-45] [INFO] [1721157459.156319144] [camera_stf]: Spinning until stopped - publishing transform
[static_transform_publisher-45] translation: ('0.000000', '0.000000', '0.000000')
[static_transform_publisher-45] rotation: ('0.499952', '-0.500000', '0.500000', '0.500048')
[static_transform_publisher-45] from 'oakd_rgb_camera_optical_frame' to 'turtlebot4/oakd_rgb_camera_frame/rgbd_camera'
[joint_state_publisher-4] [INFO] [1721157459.209475375] [joint_state_publisher]: Waiting for robot_description to be published on the robot_description topic...
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] qrc:/qml/StyleDialog.qml:112:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] qrc:/qml/StyleDialog.qml:105:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] qrc:/qml/StyleDialog.qml:98:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] qrc:qml/Main.qml:102:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] qrc:/qml/PluginMenu.qml:27:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Gazebo/GazeboDrawer.qml:242:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [Dbg] [gz.cc:161] Subscribing to [/gazebo/starting_world].
[ruby $(which ign) gazebo-1] [Dbg] [gz.cc:163] Waiting for a world to be set from the GUI...
[ruby $(which ign) gazebo-1] [Msg] Received world [warehouse.sdf] from the GUI.
[ruby $(which ign) gazebo-1] [Dbg] [gz.cc:167] Unsubscribing from [/gazebo/starting_world].
[ruby $(which ign) gazebo-1] [Msg] Ignition Gazebo Server v6.16.0
[ruby $(which ign) gazebo-1] [Msg] Loading SDF world file[/opt/ros/humble/share/turtlebot4_ignition_bringup/worlds/warehouse.sdf].
[ruby $(which ign) gazebo-1] [Msg] Serving entity system service on [/entity/system/add]
[ruby $(which ign) gazebo-1] [Dbg] [Physics.cc:804] Loaded [ignition::physics::dartsim::Plugin] from library [/usr/lib/x86_64-linux-gnu/ign-physics-5/engine-plugins/libignition-physics-dartsim-plugin.so]
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::Physics] for entity [1]
[ruby $(which ign) gazebo-1] [Msg] Create service on [/world/warehouse/create]
[ruby $(which ign) gazebo-1] [Msg] Remove service on [/world/warehouse/remove]
[ruby $(which ign) gazebo-1] [Msg] Pose service on [/world/warehouse/set_pose]
[ruby $(which ign) gazebo-1] [Msg] Pose service on [/world/warehouse/set_pose_vector]
[ruby $(which ign) gazebo-1] [Msg] Light configuration service on [/world/warehouse/light_config]
[ruby $(which ign) gazebo-1] [Msg] Physics service on [/world/warehouse/set_physics]
[ruby $(which ign) gazebo-1] [Msg] SphericalCoordinates service on [/world/warehouse/set_spherical_coordinates]
[ruby $(which ign) gazebo-1] [Msg] Enable collision service on [/world/warehouse/enable_collision]
[ruby $(which ign) gazebo-1] [Msg] Disable collision service on [/world/warehouse/disable_collision]
[ruby $(which ign) gazebo-1] [Msg] Material service on [/world/warehouse/visual_config]
[ruby $(which ign) gazebo-1] [Msg] Material service on [/world/warehouse/wheel_slip]
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::UserCommands] for entity [1]
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::SceneBroadcaster] for entity [1]
[ruby $(which ign) gazebo-1] [Msg] Loaded level [3]
[ruby $(which ign) gazebo-1] [Msg] Serving world controls on [/world/warehouse/control], [/world/warehouse/control/state] and [/world/warehouse/playback/control]
[ruby $(which ign) gazebo-1] [Msg] Serving GUI information on [/world/warehouse/gui/info]
[ruby $(which ign) gazebo-1] [Msg] World [warehouse] initialized with [default_physics] physics profile.
[ruby $(which ign) gazebo-1] [Msg] Serving world SDF generation service on [/world/warehouse/generate_world_sdf]
[ruby $(which ign) gazebo-1] [Msg] Serving world names on [/gazebo/worlds]
[ruby $(which ign) gazebo-1] [Msg] Resource path add service on [/gazebo/resource_paths/add].
[ruby $(which ign) gazebo-1] [Msg] Resource path get service on [/gazebo/resource_paths/get].
[ruby $(which ign) gazebo-1] [Msg] Resource path resolve service on [/gazebo/resource_paths/resolve].
[create-8] [INFO] [1721157460.211562135] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157460.311625477] [ros_gz_sim]: Waiting messages on topic [robot_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Scene3D.cc:2941] The GzScene3D plugin is deprecated on v6 and will be removed on v7. Use MinimalScene together with other plugins as needed.
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/GzScene3D/GzScene3D.qml:108:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/GzScene3D/GzScene3D.qml:63:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/WorldControl/WorldControl.qml:30:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [Msg] [Msg] Ignition Gazebo GUI    v6.16.0
[ruby $(which ign) gazebo-1] [Dbg] [Gui.cc:253] Waiting for subscribers to [/gazebo/starting_world]...
[ruby $(which ign) gazebo-1] [Dbg] [Application.cc:92] Initializing application.
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [Application.cc:555] Create main window
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [PathManager.cc:66] Requesting resource paths through [/gazebo/resource_paths/get]
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [Gui.cc:333] GUI requesting list of world names. The server may be busy downloading resources. Please be patient.
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [PathManager.cc:55] Received resource paths.
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [GuiRunner.cc:145] Requesting initial state from [/world/warehouse/state]...
[ruby $(which ign) gazebo-1] [GUI] [Msg] Loading config [/opt/ros/humble/share/turtlebot4_ignition_bringup/gui/standard/gui.config]
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [Application.cc:426] Loading plugin [GzScene3D]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Video recorder stats topic advertised on [/gui/record_video/stats]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Transform mode service on [/gui/transform_mode]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Record video service on [/gui/record_video]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Move to service on [/gui/move_to]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Follow service on [/gui/follow]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Follow offset service on [/gui/follow/offset]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View angle service on [/gui/view_angle]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Move to pose service on [/gui/move_to/pose]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Camera pose topic advertised on [/gui/camera/pose]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View as transparent service on [/gui/view/transparent]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View center of mass service on [/gui/view/com]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View inertia service on [/gui/view/inertia]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View joints service on [/gui/view/joints]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View wireframes service on [/gui/view/wireframes]
[ruby $(which ign) gazebo-1] [GUI] [Msg] View collisions service on [/gui/view/collisions]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Camera view controller topic advertised on [/gui/camera/view_control]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Added plugin [3D View] to main window
[ruby $(which ign) gazebo-1] [GUI] [Msg] Loaded plugin [GzScene3D] from path [/usr/lib/x86_64-linux-gnu/ign-gazebo-6/plugins/gui/libGzScene3D.so]
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [Application.cc:426] Loading plugin [WorldControl]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Using world control service [/world/warehouse/control]
[ruby $(which ign) gazebo-1] [GUI] [Msg] Listening to stats on [/world/warehouse/stats]
[ruby $(which ign) gazebo-1] [GUI] [Dbg] [WorldControl.cc:248] Using a service to share WorldControl msgs with the server
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/TransformControl/TransformControl.qml:104:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/TransformControl/TransformControl.qml:99:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/TransformControl/TransformControl.qml:94:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/TransformControl/TransformControl.qml:89:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Turtlebot4Hmi/Turtlebot4Hmi.qml:36:3: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Turtlebot4Hmi/Turtlebot4Hmi.qml:388: ReferenceError: control is not defined
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Turtlebot4Hmi/Turtlebot4Hmi.qml:410: ReferenceError: control is not defined
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Turtlebot4Hmi/Turtlebot4Hmi.qml:432: ReferenceError: control is not defined
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Turtlebot4Hmi/Turtlebot4Hmi.qml:454: ReferenceError: control is not defined
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/Turtlebot4Hmi/Turtlebot4Hmi.qml:102:7: Unable to assign [undefined] to QString
[ruby $(which ign) gazebo-1] [Wrn] [OBJLoader.cc:91] Both `d` and `Tr` parameters defined for "Chair". Use the value of `d` for dissolve (line 8 in .mtl.)
[ruby $(which ign) gazebo-1] 
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] Skipping serialization / deserialization for models with //pose/@relative_to attribute.
[ruby $(which ign) gazebo-1] [Wrn] [Component.hh:144] Trying to serialize component with data type [N3sdf3v125WorldE], which doesn't have `operator<<`. Component will not be serialized.
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Application.cc:797] [QT] file::/WorldStats/WorldStats.qml:53:3: QML RowLayout: Binding loop detected for property "x"
[ruby $(which ign) gazebo-1] 
[ruby $(which ign) gazebo-1] [Wrn] [Model.hh:69] [GUI] [Wrn] [Component.hh:189] Trying to deserialize component with data type [N3sdf3v125WorldE], which doesn't have `operator>>`. Component will not be deserialized.
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[create-8] [INFO] [1721157461.211783467] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[create-7] [INFO] [1721157461.311750001] [ros_gz_sim]: Waiting messages on topic [robot_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[spawner-32] [INFO] [1721157461.419615045] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[create-8] [INFO] [1721157462.212143148] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[create-7] [INFO] [1721157462.312264330] [ros_gz_sim]: Waiting messages on topic [robot_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[create-8] [INFO] [1721157463.212369203] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[create-7] [INFO] [1721157463.312415189] [ros_gz_sim]: Waiting messages on topic [robot_description].
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[spawner-32] [INFO] [1721157463.427882309] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Model.hh:98] Unable to deserialize sdf::Model
[ruby $(which ign) gazebo-1] libEGL warning: egl: failed to create dri2 screen
[ruby $(which ign) gazebo-1] libEGL warning: egl: failed to create dri2 screen
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Ogre2Camera.cc:390] Ogre2Camera::SetVisibilityMask: Mask bits c0000000 are set but will be ignored as they conflict with the reserved bits used internally by the ogre2 backend.
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [OBJLoader.cc:91] Both `d` and `Tr` parameters defined for "Chair". Use the value of `d` for dissolve (line 8 in .mtl.)
[ruby $(which ign) gazebo-1] 
[create-8] [INFO] [1721157464.212529701] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157464.312568345] [ros_gz_sim]: Waiting messages on topic [robot_description].
[create-8] [INFO] [1721157465.212686289] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157465.312699089] [ros_gz_sim]: Waiting messages on topic [robot_description].
[spawner-32] [INFO] [1721157465.436815384] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
[create-8] [INFO] [1721157466.212846090] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157466.312863658] [ros_gz_sim]: Waiting messages on topic [robot_description].
[create-8] [INFO] [1721157467.213011678] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157467.313053854] [ros_gz_sim]: Waiting messages on topic [robot_description].
[spawner-32] [INFO] [1721157467.446630593] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
[create-8] [INFO] [1721157468.213171513] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157468.313198341] [ros_gz_sim]: Waiting messages on topic [robot_description].
[create-8] [INFO] [1721157469.213340077] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157469.313350638] [ros_gz_sim]: Waiting messages on topic [robot_description].
[spawner-32] [ERROR] [1721157469.457358755] [spawner_joint_state_broadcaster]: Controller manager not available
[ERROR] [spawner-32]: process has died [pid 65510, exit code 1, cmd '/opt/ros/humble/lib/controller_manager/spawner joint_state_broadcaster -c controller_manager --ros-args -r __ns:=/'].
[INFO] [spawner-46]: process started with pid [65889]
[create-8] [INFO] [1721157470.213512309] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157470.313593117] [ros_gz_sim]: Waiting messages on topic [robot_description].
[create-8] [INFO] [1721157471.213695026] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157471.313773565] [ros_gz_sim]: Waiting messages on topic [robot_description].
[spawner-46] [INFO] [1721157472.102644909] [spawner_diffdrive_controller]: Waiting for '/controller_manager' node to exist
[create-8] [INFO] [1721157472.213860706] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].

[spawner-46] [INFO] [1721157474.112844972] [spawner_diffdrive_controller]: Waiting for '/controller_manager' node to exist
[create-8] [INFO] [1721157474.214332214] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[create-7] [INFO] [1721157474.314178854] [ros_gz_sim]: Waiting messages on topic [robot_description].

[create-7] [INFO] [1721157477.383824470] [ros_gz_sim]: Requested creation of entity.
[create-7] [INFO] [1721157477.383949578] [ros_gz_sim]: OK creation of entity.
[INFO] [create-7]: process has finished cleanly [pid 65121]
[ruby $(which ign) gazebo-1] [INFO] [1721157477.814461715] [GazeboSimROS2ControlPlugin]: robot_param_node is robot_state_publisher
[ruby $(which ign) gazebo-1] [INFO] [1721157477.814510858] [GazeboSimROS2ControlPlugin]: robot_param_node is robot_description
[ruby $(which ign) gazebo-1] [INFO] [1721157477.870819033] [gz_ros2_control]: [ign_ros2_control] Fixed joint [bumper_joint] (Entity=219)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.870989646] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_caster_joint] (Entity=220)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871006067] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_left_bottom_weight_block_joint] (Entity=221)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871017185] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_left_top_weight_block_joint] (Entity=222)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871028482] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_right_bottom_weight_block_joint] (Entity=223)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871041305] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_right_top_weight_block_joint] (Entity=224)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871052887] [gz_ros2_control]: [ign_ros2_control] Fixed joint [imu_joint] (Entity=225)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871067138] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_front_center_left_joint] (Entity=226)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871079764] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_front_center_right_joint] (Entity=227)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871091795] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_front_left_joint] (Entity=228)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871105944] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_front_right_joint] (Entity=229)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871116931] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_left_joint] (Entity=230)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871127789] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_right_joint] (Entity=231)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871140650] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_intensity_side_left_joint] (Entity=232)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871156231] [gz_ros2_control]: [ign_ros2_control] Fixed joint [ir_omni_joint] (Entity=233)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871168912] [gz_ros2_control]: [ign_ros2_control] Fixed joint [mouse_joint] (Entity=234)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871181047] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_left_tower_standoff_joint] (Entity=235)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871193075] [gz_ros2_control]: [ign_ros2_control] Fixed joint [front_right_tower_standoff_joint] (Entity=236)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871205348] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_camera_bracket_joint] (Entity=237)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871217235] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_joint] (Entity=238)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871227547] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_imu_joint] (Entity=239)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871243135] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_left_camera_joint] (Entity=240)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871253635] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_left_camera_optical_joint] (Entity=241)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871268035] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_rgb_camera_joint] (Entity=242)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871280505] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_rgb_camera_optical_joint] (Entity=243)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871292619] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_right_camera_joint] (Entity=244)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871305746] [gz_ros2_control]: [ign_ros2_control] Fixed joint [oakd_right_camera_optical_joint] (Entity=245)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871319604] [gz_ros2_control]: [ign_ros2_control] Fixed joint [rear_left_tower_standoff_joint] (Entity=246)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871332497] [gz_ros2_control]: [ign_ros2_control] Fixed joint [rear_right_tower_standoff_joint] (Entity=247)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871341570] [gz_ros2_control]: [ign_ros2_control] Fixed joint [rplidar_joint] (Entity=248)] is skipped
[ruby $(which ign) gazebo-1] [INFO] [1721157477.871355707] [gz_ros2_control]: [ign_ros2_control] Fixed joint [tower_sensor_plate_joint] (Entity=249)] is skipped
[spawner-46] [INFO] [1721157478.139822216] [spawner_diffdrive_controller]: Waiting for '/controller_manager' node to exist
[create-8] [INFO] [1721157478.214950362] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ruby $(which ign) gazebo-1] [ERROR] [1721157478.374403021] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[ruby $(which ign) gazebo-1] [ERROR] [1721157478.874573690] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[create-8] [INFO] [1721157479.215228533] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ruby $(which ign) gazebo-1] [ERROR] [1721157479.374756620] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[ruby $(which ign) gazebo-1] [ERROR] [1721157479.874908874] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[spawner-46] [ERROR] [1721157480.148082876] [spawner_diffdrive_controller]: Controller manager not available
[create-8] [INFO] [1721157480.215466736] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ERROR] [spawner-46]: process has died [pid 65889, exit code 1, cmd '/opt/ros/humble/lib/controller_manager/spawner diffdrive_controller -c controller_manager --ros-args -r __ns:=/ --params-file /opt/ros/humble/share/irobot_create_control/config/control.yaml'].
[ruby $(which ign) gazebo-1] [ERROR] [1721157480.375066245] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[ruby $(which ign) gazebo-1] [ERROR] [1721157480.875274468] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[create-8] [INFO] [1721157481.215650766] [ros_gz_sim]: Waiting messages on topic [standard_dock_description].
[ruby $(which ign) gazebo-1] [ERROR] [1721157481.375468647] [gz_ros2_control]: robot_state_publisher service not available, waiting again...


[parameter_bridge-27] [INFO] [1721157485.889760744] [hmi_led_msg_bridge]: Passing message from ROS std_msgs/msg/Int32 to Gazebo gz.msgs.Int32 (showing msg only once per type)
[ruby $(which ign) gazebo-1] [ERROR] [1721157486.379560755] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[ruby $(which ign) gazebo-1] [ERROR] [1721157486.879697903] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[parameter_bridge-25] [INFO] [1721157486.889896882] [hmi_display_msg_bridge]: Passing message from ROS std_msgs/msg/String to Gazebo gz.msgs.StringMsg (showing msg only once per type)
[parameter_bridge-25] [INFO] [1721157486.890000763] [hmi_display_msg_bridge]: Passing message from ROS std_msgs/msg/Int32 to Gazebo gz.msgs.Int32 (showing msg only once per type)


[ruby $(which ign) gazebo-1] [ERROR] [1721157493.883080619] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[ruby $(which ign) gazebo-1] [ERROR] [1721157494.384320617] [gz_ros2_control]: robot_state_publisher service not available, waiting again...
[ruby $(which ign) gazebo-1] [INFO] [1721157494.884764628] [gz_ros2_control]: connected to service!! robot_state_publisher asking for robot_description
[ruby $(which ign) gazebo-1] [INFO] [1721157494.887072461] [gz_ros2_control]: Received URDF from param server
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895316222] [gz_ros2_control]: The position_proportional_gain has been set to: 0.1
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895691325] [gz_ros2_control]: Loading joint: left_wheel_joint
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895761087] [gz_ros2_control]: 	State:
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895777155] [gz_ros2_control]: 		 velocity
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895789335] [gz_ros2_control]: 		 position
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895799214] [gz_ros2_control]: 	Command:
[ruby $(which ign) gazebo-1] [INFO] [1721157494.895808081] [gz_ros2_control]: 		 velocity
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896039779] [gz_ros2_control]: Loading sensor: imu
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896056841] [gz_ros2_control]: 	State:
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896094475] [resource_manager]: Initialize hardware 'left_wheel_controller' 
[ruby $(which ign) gazebo-1] [WARN] [1721157494.896118113] [gz_ros2_control]: On init...
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896192632] [resource_manager]: Successful initialization of hardware 'left_wheel_controller'
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896240745] [resource_manager]: 'configure' hardware 'left_wheel_controller' 
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896249061] [gz_ros2_control]: System Successfully configured!
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896263990] [resource_manager]: Successful 'configure' of hardware 'left_wheel_controller'
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896272629] [resource_manager]: 'activate' hardware 'left_wheel_controller' 
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896280577] [resource_manager]: Successful 'activate' of hardware 'left_wheel_controller'
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896425105] [gz_ros2_control]: The position_proportional_gain has been set to: 0.1
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896477799] [gz_ros2_control]: Loading joint: right_wheel_joint
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896490347] [gz_ros2_control]: 	State:
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896499954] [gz_ros2_control]: 		 velocity
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896509601] [gz_ros2_control]: 		 position
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896518848] [gz_ros2_control]: 	Command:
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896527788] [gz_ros2_control]: 		 velocity
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896967311] [gz_ros2_control]: Loading sensor: imu
[ruby $(which ign) gazebo-1] [INFO] [1721157494.896994740] [gz_ros2_control]: 	State:
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897009434] [resource_manager]: Initialize hardware 'right_wheel_controller' 
[ruby $(which ign) gazebo-1] [WARN] [1721157494.897017053] [gz_ros2_control]: On init...
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897041748] [resource_manager]: Successful initialization of hardware 'right_wheel_controller'
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897059733] [resource_manager]: 'configure' hardware 'right_wheel_controller' 
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897203617] [gz_ros2_control]: System Successfully configured!
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897224966] [resource_manager]: Successful 'configure' of hardware 'right_wheel_controller'
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897234788] [resource_manager]: 'activate' hardware 'right_wheel_controller' 
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897243163] [resource_manager]: Successful 'activate' of hardware 'right_wheel_controller'
[ruby $(which ign) gazebo-1] [INFO] [1721157494.897250592] [gz_ros2_control]: Loading controller_manager
[ruby $(which ign) gazebo-1] [Wrn] [Component.hh:127] Trying to serialize component with data type [St10shared_ptrIN3sdf3v127ElementEE], which doesn't have `operator<<`. Component will not be serialized.
[ruby $(which ign) gazebo-1] [GUI] [Msg] Added plugin 32mResource paths published on [/gazebo/resource_paths].
[ruby $(which ign) gazebo-1] [Msg] Server control service on [/server_control].
[ruby $(which ign) gazebo-1] [Msg] Found no publishers on /stats, adding root stats topic
[ruby $(which ign) gazebo-1] [Msg] Found no publishers on /clock, adding root clock topic
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:496] Creating PostUpdate worker threads: 2
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (0)
[ruby $(which ign) gazebo-1] [Msg] Serving scene information on [/world/warehouse/scene/info]
[ruby $(which ign) gazebo-1] [Msg] Serving graph information on [/world/warehouse/scene/graph]
[ruby $(which ign) gazebo-1] [Msg] Serving full state on [/world/warehouse/state]
[ruby $(which ign) gazebo-1] [Msg] Serving full state (async) on [/world/warehouse/state_async]
[ruby $(which ign) gazebo-1] [Msg] Publishing scene information on [/world/warehouse/scene/info]
[ruby $(which ign) gazebo-1] [Msg] Publishing entity deletions on [/world/warehouse/scene/deletion]
[ruby $(which ign) gazebo-1] [Msg] Publishing state changes on [/world/warehouse/state]
[ruby $(which ign) gazebo-1] [Msg] Publishing pose messages on [/world/warehouse/pose/info]
[ruby $(which ign) gazebo-1] [Msg] Publishing dynamic pose messages on [/world/warehouse/dynamic_pose/info]
[ruby $(which ign) gazebo-1] [Dbg] [EntityComponentManager.cc:1623] Updated state thread iterators: 8 threads processing around 17 entities each.
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ign_ros2_control::IgnitionROS2ControlPlugin] for entity [136]
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::PosePublisher] for entity [136]
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::Contact] for entity [136]
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:506] Configuring Sensors system
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:431] SensorsPrivate::Run
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::Sensors] for entity [136]
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:411] SensorsPrivate::RenderThread started
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:258] Waiting for init
[ruby $(which ign) gazebo-1] [Dbg] [UserCommands.cc:1303] Created entity [136] named [turtlebot4]
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:523] Exiting postupdate worker thread (0)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:496] Creating PostUpdate worker threads: 6
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (0)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (1)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (2)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (3)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (4)
[ruby $(which ign) gazebo-1] [WARN] [1721157495.312802275] [gz_ros2_control]:  Desired controller update period (0.001 s) is slower than the gazebo simulation period (0 s).
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [Component.hh:172] Trying to deserialize component with data type [St10shared_ptrIN3sdf3v127ElementEE], which doesn't have `operator>>`. Component will not be deserialized.
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [ColladaLoader.cc:2394] Triangle input semantic: 'COLOR' is currently not supported
[ruby $(which ign) gazebo-1] [GUI] [Wrn] [ColladaLoader.cc:2394] Triangle input semantic: 'COLOR' is currently not supported
[ruby $(which ign) gazebo-1] [Wrn] [ColladaLoader.cc:2394] Triangle input semantic: 'COLOR' is currently not supported
[ruby $(which ign) gazebo-1] [Wrn] [ColladaLoader.cc:2394] Triangle input semantic: 'COLOR' is currently not supported
[ruby $(which ign) gazebo-1] [Dbg] [SystemManager.cc:70] Loaded system [ignition::gazebo::systems::PosePublisher] for entity [254]
[ruby $(which ign) gazebo-1] [Dbg] [UserCommands.cc:1303] Created entity [254] named [standard_dock]
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:610] Initialization needed
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:270] Initializing render context
[ruby $(which ign) gazebo-1] [Msg] Loading plugin [ignition-rendering-ogre]
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:523] Exiting postupdate worker thread (4[Dbg] [SimulationRunner.cc:523] 4[Dbg] [SimulationRunner.cc:523] )[Dbg] [SimulationRunner.cc:523] 
[ruby $(which ign) gazebo-1] Exiting postupdate worker thread (2)
[ruby $(which ign) gazebo-1] 3)
[ruby $(which ign) gazebo-1] Exiting postupdate worker thread (1[Dbg] [SimulationRunner.cc:523] Exiting postupdate worker thread (0)
[ruby $(which ign) gazebo-1] 1)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:496] Creating PostUpdate worker threads: 7
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (0)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (1)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (2)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (3)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (4)
[ruby $(which ign) gazebo-1] [Dbg] [SimulationRunner.cc:507] Creating postupdate worker thread (5)
[ruby $(which ign) gazebo-1] [Dbg] [RenderUtil.cc:2591] Create scene [scene]
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:610] Initialization needed
[ruby $(which ign) gazebo-1] [Dbg] [Sensors.cc:284] Rendering Thread initialized
[ruby $(which ign) gazebo-1] [Dbg] [RgbdCameraSensor.cc:216] RGB images for [turtlebot4::oakd_rgb_camera_frame::rgbd_camera] advertised on [world/warehouse/model/turtlebot4/link/oakd_rgb_camera_frame/sensor/rgbd_camera/image]
[ruby $(which ign) gazebo-1] [Dbg] [RgbdCameraSensor.cc:230] Depth images for [turtlebot4::oakd_rgb_camera_frame::rgbd_camera] advertised on [world/warehouse/model/turtlebot4/link/oakd_rgb_camera_frame/sensor/rgbd_camera/depth_image]
[ruby $(which ign) gazebo-1] [Dbg] [RgbdCameraSensor.cc:244] Points for [turtlebot4::oakd_rgb_camera_frame::rgbd_camera] advertised on [world/warehouse/model/turtlebot4/link/oakd_rgb_camera_frame/sensor/rgbd_camera/points]
[ruby $(which ign) gazebo-1] [Dbg] [CameraSensor.cc:760] Camera info for [turtlebot4::oakd_rgb_camera_frame::rgbd_camera] advertised on [world/warehouse/model/turtlebot4/link/oakd_rgb_camera_frame/sensor/rgbd_camera/camera_info]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::base_link::cliff_front_left] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_front_left/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::base_link::cliff_front_left] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_front_left/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::base_link::cliff_front_right] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_front_right/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::base_link::cliff_front_right] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_front_right/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::base_link::cliff_side_left] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_side_left/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::base_link::cliff_side_left] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_side_left/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::base_link::cliff_side_right] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_side_right/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::base_link::cliff_side_right] advertised on [world/warehouse/model/turtlebot4/link/base_link/sensor/cliff_side_right/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_front_center_left::ir_intensity_front_center_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_center_left/sensor/ir_intensity_front_center_left/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_front_center_left::ir_intensity_front_center_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_center_left/sensor/ir_intensity_front_center_left/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_front_center_right::ir_intensity_front_center_right] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_center_right/sensor/ir_intensity_front_center_right/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_front_center_right::ir_intensity_front_center_right] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_center_right/sensor/ir_intensity_front_center_right/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_front_left::ir_intensity_front_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_left/sensor/ir_intensity_front_left/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_front_left::ir_intensity_front_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_left/sensor/ir_intensity_front_left/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_front_right::ir_intensity_front_right] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_right/sensor/ir_intensity_front_right/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_front_right::ir_intensity_front_right] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_front_right/sensor/ir_intensity_front_right/scan/points]
[ruby $(which ign) gazebo-1] [Wrn] [OgreGpuRays.cc:258] Horizontal FOV for GPU rays is capped at 180 degrees.
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_left::ir_intensity_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_left/sensor/ir_intensity_left/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_left::ir_intensity_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_left/sensor/ir_intensity_left/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_right::ir_intensity_right] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_right/sensor/ir_intensity_right/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_right::ir_intensity_right] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_right/sensor/ir_intensity_right/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::ir_intensity_side_left::ir_intensity_side_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_side_left/sensor/ir_intensity_side_left/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::ir_intensity_side_left::ir_intensity_side_left] advertised on [world/warehouse/model/turtlebot4/link/ir_intensity_side_left/sensor/ir_intensity_side_left/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [turtlebot4::rplidar_link::rplidar] advertised on [world/warehouse/model/turtlebot4/link/rplidar_link/sensor/rplidar/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [turtlebot4::rplidar_link::rplidar] advertised on [world/warehouse/model/turtlebot4/link/rplidar_link/sensor/rplidar/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [standard_dock::green_buoy_link::green_buoy] advertised on [world/warehouse/model/standard_dock/link/green_buoy_link/sensor/green_buoy/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [standard_dock::green_buoy_link::green_buoy] advertised on [world/warehouse/model/standard_dock/link/green_buoy_link/sensor/green_buoy/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [standard_dock::halo_link::halo] advertised on [world/warehouse/model/standard_dock/link/halo_link/sensor/halo/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [standard_dock::halo_link::halo] advertised on [world/warehouse/model/standard_dock/link/halo_link/sensor/halo/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [standard_dock::red_buoy_link::red_buoy] advertised on [world/warehouse/model/standard_dock/link/red_buoy_link/sensor/red_buoy/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [standard_dock::red_buoy_link::red_buoy] advertised on [world/warehouse/model/standard_dock/link/red_buoy_link/sensor/red_buoy/scan/points]
[ruby $(which ign) gazebo-1] [Dbg] [Lidar.cc:132] Laser scans for [standard_dock::yellow_buoy_link::yellow_buoy] advertised on [world/warehouse/model/standard_dock/link/yellow_buoy_link/sensor/yellow_buoy/scan]
[ruby $(which ign) gazebo-1] [Dbg] [GpuLidarSensor.cc:163] Lidar points for [standard_dock::yellow_buoy_link::yellow_buoy] advertised on [world/warehouse/model/standard_dock/link/yellow_buoy_link/sensor/yellow_buoy/scan/points]
[parameter_bridge-9] [INFO] [1721157505.543785524] [cmd_vel_bridge]: Passing message from ROS geometry_msgs/msg/Twist to Gazebo gz.msgs.Twist (showing msg only once per type)
[turtlebot4_node-30] [INFO] [1721157505.671467259] [turtlebot4_node]: OAKD stopped
[turtlebot4_node-30] [INFO] [1721157505.671534700] [turtlebot4_node]: RPLIDAR stopped
[turtlebot4_node-30] [INFO] [1721157520.280092031] [turtlebot4_node]: Setting EStop
[turtlebot4_node-30] [INFO] [1721157521.281088836] [turtlebot4_node]: e_stop service available, sending request
[motion_control-35] [INFO] [1721157521.281749204] [motion_control]: E-Stop request: Set system E-Stop on, cutting motor power
[turtlebot4_node-30] [INFO] [1721157521.281970178] [turtlebot4_node]: e_stop service got results: Success
[turtlebot4_node-30] [INFO] [1721157521.282000746] [turtlebot4_node]: Set system E-Stop on, cutting motor power
[turtlebot4_node-30] [ERROR] [1721157527.110090867] [turtlebot4_node]: Power OFF
[turtlebot4_node-30] [INFO] [1721157528.110718692] [turtlebot4_node]: robot_power service available, sending request
[motion_control-35] [ERROR] [1721157528.111243075] [motion_control]: Power off request: Set system power off failed (not supported in sim)
[turtlebot4_node-30] [INFO] [1721157528.111973619] [turtlebot4_node]: robot_power service got results: Failed
[turtlebot4_node-30] [INFO] [1721157528.112127419] [turtlebot4_node]: Set system power off failed (not supported in sim)
[turtlebot4_node-30] [INFO] [1721157530.630497445] [turtlebot4_node]: Clearing EStop
[turtlebot4_node-30] [INFO] [1721157531.632158074] [turtlebot4_node]: e_stop service available, sending request
[turtlebot4_node-30] [INFO] [1721157531.635573658] [turtlebot4_node]: e_stop service got results: Success
[turtlebot4_node-30] [INFO] [1721157531.635636381] [turtlebot4_node]: Set system E-Stop off, enabling motor power
[motion_control-35] [INFO] [1721157531.636356844] [motion_control]: E-Stop request: Set system E-Stop off, enabling motor power
[turtlebot4_node-30] [ERROR] [1721157535.682659323] [turtlebot4_node]: Service oakd/stop_camera unavailable.
[turtlebot4_node-30] [ERROR] [1721157535.693298306] [turtlebot4_node]: Service stop_motor unavailable.
[turtlebot4_node-30] [ERROR] [1721157545.240502696] [turtlebot4_node]: Power OFF
[turtlebot4_node-30] [INFO] [1721157546.241072663] [turtlebot4_node]: robot_power service available, sending request
[motion_control-35] [ERROR] [1721157546.241380744] [motion_control]: Power off request: Set system power off failed (not supported in sim)
[turtlebot4_node-30] [INFO] [1721157546.241802533] [turtlebot4_node]: robot_power service got results: Failed
[turtlebot4_node-30] [INFO] [1721157546.241876548] [turtlebot4_node]: Set system power off failed (not supported in sim)
[turtlebot4_node-30] [ERROR] [1721157551.290799167] [turtlebot4_node]: Power OFF
[turtlebot4_node-30] [INFO] [1721157552.291417463] [turtlebot4_node]: robot_power service available, sending request
[motion_control-35] [ERROR] [1721157552.292213818] [motion_control]: Power off request: Set system power off failed (not supported in sim)
[turtlebot4_node-30] [INFO] [1721157552.293008769] [turtlebot4_node]: robot_power service got results: Failed
[turtlebot4_node-30] [INFO] [1721157552.293191465] [turtlebot4_node]: Set system power off failed (not supported in sim)
[turtlebot4_node-30] [INFO] [1721157558.820561726] [turtlebot4_node]: Undocking
[turtlebot4_node-30] [INFO] [1721157558.820631063] [turtlebot4_node]: Waiting for undock action server
[turtlebot4_node-30] [INFO] [1721157559.821180905] [turtlebot4_node]: undock action server available, sending goal
[turtlebot4_node-30] [ERROR] [1721157559.821843092] [turtlebot4_node]: undock goal was rejected by server
[turtlebot4_node-30] [ERROR] [1721157559.822062458] [turtlebot4_node.rclcpp_action]: unknown goal response, ignoring...
[turtlebot4_node-30] [ERROR] [1721157559.822144260] [turtlebot4_node.rclcpp_action]: unknown goal response, ignoring...
[motion_control-35] [INFO] [1721157559.821961064] [motion_control]: Received new undock goal
```

### To Reproduce

`ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py`

**Comment by aharshac** [comment]
Full log: [tb4_sim_error.log](https://github.com/user-attachments/files/16255693/tb4_sim_error.log)

**Comment by RustyCPR** [comment]
Hi @aharshac 
Can you please detail the hardware specs of your PC you are running this on? 
It seems like the system does not have the required resources (CPU, RAM etc.) to run this simulation.
Best regards,
Rusty

**Comment by aharshac** [comment]
Hello @RustyCPR 

I've seen this issue on two PCs. One with 16 GB RAM, 4GB 1050Ti and another with 32 GB RAM, 4 GB P2000. Both of them were running local Fast DDS discovery servers. This problem exists with *Create3 Ignition* as well, but not with *Gazebo Classic*.

I did not see this problem when I switched to Cyclone DDS with simple discovery. I want to use Turtlebot4 sim with multiple robots for frontier exploration, and I was unable to spawn more than one robot with Cyclone.

I think this issue is caused by *ros2_control* https://github.com/ros-controls/ros2_control/issues/1200. There's a fix https://github.com/ros-controls/ros2_control/pull/1562, but I was unable to build the branch locally.

Here's how I source my env
https://gist.github.com/aharshac/b07929459d00935bf6b0259144202e58 

Thanks

**Comment by aharshac** [comment]
@RustyCPR Can you confirm if I'm sourcing my environment properly?
This is how I start the discovery serer
```sh
#!/bin/bash
source /opt/ros/humble/setup.bash
fastdds discovery -i 0 -p 11811
```

fastdds_super_client.xml
```xml
<?xml version="1.0" encoding="UTF-8" ?>
 <dds>
     <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
         <participant profile_name="super_client_profile" is_default_profile="true">
             <rtps>
                 <builtin>
                     <discovery_config>
                         <discoveryProtocol>SUPER_CLIENT</discoveryProtocol>
                         <discoveryServersList>
                             <RemoteServer prefix="44.53.00.5f.45.50.52.4f.53.49.4d.41">
                                 <metatrafficUnicastLocatorList>
                                     <locator>
                                         <udpv4>
                                             <address>127.0.0.1</address>
                                             <port>11811</port>
                                         </udpv4>
                                     </locator>
                                 </metatrafficUnicastLocatorList>
                             </RemoteServer>
                         </discoveryServersList>
                     </discovery_config>
                 </builtin>
             </rtps>
         </participant>
     </profiles>
 </dds>
```

**Comment by aharshac** [comment]
https://github.com/iRobotEducation/create3_sim/issues/194#issuecomment-2245957234

**Comment by destogl** [comment]
> Hello @RustyCPR
> 
> I've seen this issue on two PCs. One with 16 GB RAM, 4GB 1050Ti and another with 32 GB RAM, 4 GB P2000. Both of them were running local Fast DDS discovery servers. This problem exists with _Create3 Ignition_ as well, but not with _Gazebo Classic_.
> 
> I did not see this problem when I switched to Cyclone DDS with simple discovery. I want to use Turtlebot4 sim with multiple robots for frontier exploration, and I was unable to spawn more than one robot with Cyclone.
> 
> I think this issue is caused by _ros2_control_ [ros-controls/ros2_control#1200](https://github.com/ros-controls/ros2_control/issues/1200). There's a fix [ros-controls/ros2_control#1562](https://github.com/ros-controls/ros2_control/pull/1562), but I was unable to build the branch locally.
> 
> Here's how I source my env https://gist.github.com/aharshac/b07929459d00935bf6b0259144202e58
> 
> Thanks

We have fixed the issues in all supported distros. Currently in source, but feel free to test and give us feedback!

**Comment by RustyCPR** [comment]
Hi @aharshac 

Please try removing/unsetting the FASTRTPS_DEFAULT_PROFILES_FILE from your sourcing script. 
That profile is just pointing it towards the server you started and setting it as super client which is a duplication of the other environment variables that you have set.

**Comment by aharshac** [comment]
@RustyCPR The simulator works as expected after https://github.com/ros-controls/ros2_control/pull/1562 was merged. Thanks @destogl for the fix.
